wsinfer/cli/infer.py

```diff
@@ -374,3 +375,6 @@ def run(
         json.dump(run_metadata, f, indent=2)
 
     click.secho("Finished.", fg="green")
+
+    csvs = list((results_dir / "model-outputs-csv").glob("*.csv"))
+    write_geojsons(csvs, results_dir, num_workers)
```

Comment:
the geojson writing should happen in the fille `run_inference.py` -- see https://github.com/SBU-BMI/wsinfer/blob/123d7c1aaa8bf72b00e857a090db6385db069720/wsinfer/modellib/run_inference.py#L194 for where the CSVs are written. The GeoJSON conversion can happen right after that.